cmake/compiler/gcc.cmake

```diff
@@ -1,10 +1,18 @@
+if("${ZEPHYR_TOOLCHAIN_VARIANT}" STREQUAL "zephyr")
+  set(cc "gcc")
```

Comment:
In the zephyr build system it would be more common to have an uppercase name for 'cc'.

Reply:
Will make this change.